package.json

```diff
@@ -10,8 +10,8 @@
     "url": "git@github.com:bcoe/c8.git"
   },
   "scripts": {
-    "test": "node ./bin/c8.js mocha --timeout=8000 ./test/*.js",
-    "coverage": "node ./bin/c8.js --check-coverage mocha --timeout=8000 ./test/*.js",
+    "test": "TS_NODE_SKIP_PROJECT=true node ./bin/c8.js mocha --timeout=8000 ./test/*.js",
```

Comment:
You can `npm i --save-dev cross-env` to fix the Windows tests:

https://www.npmjs.com/search?q=cross-env

👍

Reply:
Done!